lib/msf/core/payload/windows/x64/reverse_http_x64.rb

```diff
@@ -448,7 +455,7 @@ def asm_reverse_http(opts={})
         push 0x40
         pop rdx
         mov r9, rdx                   ; flProtect (0x40=PAGE_EXECUTE_READWRITE)
-        shl edx, 16                   ; dwSize
+        shl edx, #{"0x%.8x" % shift_size} ; dwSize
```

Comment:
not a blocker: I wonder why this uses shl versus inlining the size similar to the reverse_http 🤔

Reply:
Yeah was curious myself, but just adjusted whatever was already there! :)